### DISCUSSION TITLE
RDF4J 4.3.7

### DISCUSSION THREAD

**Discussion by hmottestad** [eclipse-rdf4j/rdf4j#4804] [Announcements]
RDF4J 4.3.7 is now available. This is a patch release fixing 6 bugs.

For more details, have a look at the [release notes](https://rdf4j.org/release-notes/4.3.7).

Links
[Download RDF4J](https://rdf4j.org/download/)
[release notes](https://rdf4j.org/release-notes/4.3.7).

<hr /><em>This discussion was created from the release <a href='https://github.com/eclipse-rdf4j/rdf4j/releases/tag/4.3.7'>RDF4J 4.3.7</a>.</em>

**Comment by reckart** [comment]
It seems RDF4J 4.3.7 has not arrived on Maven Central yet - did anything get stuck?

**Reply by hmottestad** [reply]
Ahh, yeah, just saw that the jenkins failed due to a connection issue with nexus sonatype...it's unfortunately quite frequent. I've triggered a rebuild.

**Reply by hmottestad** [reply]
Should be available now: https://central.sonatype.com/artifact/org.eclipse.rdf4j/rdf4j-bom/4.3.7
